app/src/main/java/net/skyscanner/backpack/demo/stories/ColoredCalendarStory.kt

```diff
@@ -14,6 +14,7 @@ class ColoredCalendarStory : Story() {
 
     val controller = ExampleBpkCalendarController(requireContext())
     val bpkCalendar = view.findViewById<BpkCalendar>(R.id.bpkCalendar)
+    bpkCalendar.isSelectionTypeSwitcherVisible = true
```

Comment:
Instead of this line we can replace it with adding a switcher in the example page and here adding a click/change listener that changes the selection/recreate the controller

Reply:
I got this now. 
I am fine with that.